cmd/autoscaler/main.go

```diff
@@ -78,12 +78,13 @@ func main() {
 	log.Printf("Registering %d clients", len(injection.Default.GetClients()))
 	log.Printf("Registering %d informer factories", len(injection.Default.GetInformerFactories()))
 	log.Printf("Registering %d informers", len(injection.Default.GetInformers()))
+	log.Printf("Registering %d filtered informers", len(injection.Default.GetFilteredInformers()))
 	log.Printf("Registering %d controllers", controllerNum)
 
 	// Adjust our client's rate limits based on the number of controller's we are running.
 	cfg.QPS = controllerNum * rest.DefaultQPS
 	cfg.Burst = controllerNum * rest.DefaultBurst
-
+	ctx = context.WithValue(ctx, filteredinformerfactory.LabelKey{}, []string{serving.RevisionUID})
```

Comment:
should this use the [WithSelectors helper](https://github.com/knative/pkg/pull/1940/files#diff-2663edce6aafb6c4222afff27cf59df35d3f043c27efdf04c68db9d1e9e44dd9R108) introduced in the pkg PR (and already used elsewhere in this PR actually)?

Reply:
Thank for the reminder. Fixed.